travelrec/recommendations_pipeline.py

```diff
@@ -0,0 +1,70 @@
+from travelrec.similarity import (
+    ClimateSimilarity,
+    GeoFeaturesSimilarity,
+    ActvitySimilarity,
+)
+from travelrec.sparql import (
+    get_location,
+    get_cities_with_temperature,
+    get_geofeatures,
+    get_activities,
+)
+from travelrec.processed_statement import ProcessedStatement
+from travelrec.scoring import score_cities_parallelly
+
+search_distances = {
+    'climate': 300,
+    'geofeatures': 150,
+    'activities': 75
+}
+
+def recommendations_pipeline(query, nlp, verbose=False):
+    climateSim = ClimateSimilarity()
+    geoSim = GeoFeaturesSimilarity()
+    activitySim = ActvitySimilarity()
+
+    if verbose:
```

Comment:
Use `logging` module. Instead of `if` use `logging.debug` and just logging level based on `verbose` parameter.